DSA-Campaign-Uplift-Estimation/src/main/java/com/google/sps/servlets/KeywordCampaignsServlet.java

```diff
@@ -113,4 +113,26 @@ public static Entity createEntityFromKeywordCampaign(KeywordCampaign KeywordCamp
 
         return keywordCampaignEntity;
     }
+
+    // Retrieves a unique keyword campaign id from datastore.
+    public static String getNewKeywordCampaignId() {
```

Comment:
Aren't we doing something very similar for DSA campaign ids. If so we can probably have a function to minimize code duplication.

Reply:
Done.